src/Raven.Server/Utils/TypeConverter.cs

```diff
@@ -226,7 +226,7 @@ private static object ToBlittableSupportedType(object root, object value, bool f
 
                 return @object;
             }
-
+            // looks like best option
```

Comment:
best option considering performance / allocations right? please update the comment to reflect that

Reply:
done, sorry thats a note to myself that I forgot to delete.